x-pack/legacy/plugins/ml/public/application/datavisualizer/index_based/components/field_data_card/content_types/number_content.tsx

```diff
@@ -123,30 +137,22 @@ export const NumberContent: FC<FieldDataCardProps> = ({ config }) => {
           <DisplayValue value={kibanaFieldFormat(max, fieldFormat)} />
         </EuiFlexItem>
       </EuiFlexGroup>
-
       <EuiSpacer size="s" />
-
-      <EuiFlexGroup gutterSize="xs" justifyContent="center">
-        <EuiFlexItem grow={false} style={{ width: 200 }} className="eui-textTruncate">
-          <EuiSelect
-            options={detailsOptions}
-            value={detailsMode}
-            compressed
-            onChange={e => setDetailsMode(e.target.value as DETAILS_MODE)}
-            style={{ width: '200px' }}
-            aria-label={i18n.translate(
-              'xpack.ml.fieldDataCard.cardNumber.selectMetricDetailsDisplayAriaLabel',
-              {
-                defaultMessage: 'Select display option for metric details',
-              }
-            )}
-            data-test-subj="mlFieldDataCardNumberDetailsSelect"
-          />
-        </EuiFlexItem>
-      </EuiFlexGroup>
-
+      <EuiButtonGroup
```

Comment:
@mdefazio is there enough contrast in the colors between the two states in this control? I'm wondering if the selected state should be a darker grey to satisfy accessibility requirements?

![image](https://user-images.githubusercontent.com/7405507/72255049-a3d71000-35fd-11ea-80bf-76194d286650.png)

Reply:
It looks like it's passing AAA.  This is managed in EUI, so if there are needs to support WCAG AA, we can put an issue into EUI for it.